component/kmscrypto/kms/localkms/localkms.go

```diff
@@ -17,12 +17,16 @@ import (
 	"github.com/google/tink/go/aead"
 	"github.com/google/tink/go/keyset"
 
-	cryptoapi "github.com/hyperledger/aries-framework-go/component/kmscrypto/pkg/crypto"
-	"github.com/hyperledger/aries-framework-go/component/kmscrypto/pkg/crypto/primitive/bbs12381g2pub"
-	"github.com/hyperledger/aries-framework-go/component/kmscrypto/pkg/doc/util/jwkkid"
-	"github.com/hyperledger/aries-framework-go/component/kmscrypto/pkg/kms"
-	"github.com/hyperledger/aries-framework-go/component/kmscrypto/pkg/kms/localkms/internal/keywrapper"
-	"github.com/hyperledger/aries-framework-go/component/kmscrypto/pkg/secretlock"
+	kms2 "github.com/hyperledger/aries-framework-go/spi/kms"
```

Comment:
Would be better to rename `km2` to something else more descriptive, perhaps `spikms` (same comment applies to other places in this PR)

Reply:
done!